keps/sig-apps/19-Graduate-CronJob-to-Stable/kep.yaml

```diff
@@ -31,13 +31,13 @@ stage: stable
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.21"
+latest-milestone: "v1.22"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
   alpha: "v1.4"
   beta: "v1.8"
-  stable: "v1.21"
+  stable: "v1.22"
```

Comment:
👋 Thanks for this. However the `milestone.stable` key should remain as `"v1.21"` as it should be the *first* time it graduated to stable, in 1.21, rather than subsequent times.

Reply:
Done!